### PR TITLE
[bugfix] convert milliseconds to seconds.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- convert milliseconds to seconds.

### CODE REVIEW

File: src/utils/util.ts [hunk 1]
```diff
@@ -99,3 +105,11 @@ export const arrayToObject = <T>(array: T[]): { [key: string]: T } => {
   });
   return result;
 };
+
+export const isMillisecond = (timestamp: number) => {
```
Comment (minjaeli) [inline src/utils/util.ts:109]: 삼항연산을 거는것보다 함수가 나중에 관리가 편하지 않을까 생각이 듭니당. 

```
const normalizeTimestamp = (timestamp: number): number => {
  return timestamp.toString().length === 13 ? Math.floor(timestamp / 1000) : timestamp;
};
```

**Review by minjaeli — APPROVED** [review]
LGTM with comments! thanks!

File: src/utils/util.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/utils/util.ts:111]: it's a very helpful advice! thanks!

**Comment by jiyoung-an** [comment]
Thank you for review @minjaeli!

File: src/utils/util.ts [hunk 2]
```diff
@@ -99,3 +103,15 @@ export const arrayToObject = <T>(array: T[]): { [key: string]: T } => {
   });
   return result;
 };
+
+export const normalizeTimestamp = (timestamp: number) => {
+  return isMillisecond(timestamp) ? toSecond(timestamp) : timestamp;
+};
+
+export const isMillisecond = (timestamp: number) => {
+  return timestamp.toString().length === 13;
```
Comment (platfowner) [inline src/utils/util.ts:112]: minor: maybe >= 13 would be better?

**Review by platfowner — COMMENTED** [review]
LGTM with some minor comments.
Thanks!

**Pull request merged**